recipes/eip712/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "eip712" %}
+{% set version = "0.2.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/ApeWorX/{{ name }}/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: e388e6281048bda1a575a2ca75e6f721fa771c67979554b6937758eae52335cf
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vvv
+  number: 0
+
+requirements:
+  host:
```

Comment:
You should probably also add setuptools-scm here

Ref: https://github.com/ApeWorX/eip712/blob/ab8c6263ac3e85672d45933060356b60ca472e42/setup.py#L58

Reply:
It fails the build. I vaguely remember finding that it would require an older version that then creates a conda dependency conflict, but I don't recall the details. I just ran a try and it fails mentioning not being able to find the packageversion

Reply:
Hmm strange. Can you please push it so that I can see it in the logs?

Reply:
I think you're running into this: https://conda-forge.org/docs/maintainer/knowledge_base.html#using-setuptools-scm

Just add 

  script_env:
    - SETUPTOOLS_SCM_PRETEND_VERSION={{version}}